src/GitHubParserHook.php

```diff
@@ -43,14 +53,58 @@ private function setFields( ProcessingResult $result ) {
 		$this->fileName = $params['file']->getValue();
 		$this->repoName = $params['repo']->getValue();
 		$this->branchName = $params['branch']->getValue();
+
+		foreach ( self::$syntaxHighlightFields as $val ) {
+			if ( isset( $params[$val] ) ) {
+				$this->$val = $this->cleanField( $params[$val]->getValue() );
```

Comment:
Eh. This kind of dynamic shit is really bad style (in general, loose from mediawiki or PHP). No need for that complexity